includes/definitions.inc.php

```diff
@@ -1725,6 +1725,15 @@
 $config['os'][$os]['over'][2]['graph'] = 'device_xirrus_rssi';
 $config['os'][$os]['over'][2]['text']  = 'Signal RSSI';
 
+// Rittal CMC
+$os = 'rittal_cmc';
```

Comment:
The os name shouldn't be the company name and model unless that's the best we have. it should be the os it's running, like we have Cisco C2960, that runs IOS.

If this is the best name, can you use - instead of _